src/main/java/codeu/controller/BotHandlerInterface.java

```diff
@@ -0,0 +1,17 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+
+/**
+ * The interface which is implemented by each bot type in order to produce the desired
```

Comment:
This doesn't really tell me much except "this is an interface that will be implemented by stuff" This is mostly a comment that describes the language semantics (i.e. what an `interface` is) rather than what this interface is for.

I should be able to read this comment without knowing anything about bots in our application, and be able to understand what I need to do and how to use this interface. Something like:
> Defines interface for all bots to implement. Bots are invoked by users entering a regex-matched syntax, which may or may not pass additional arguments into implementing classes. Those implementations will return a string to be posted into the conversation that executed the command.